tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -799,10 +800,10 @@ def _test_cluster_limits(slurm_commands, partition, instance_type, max_count, cp
     )
 
 
-def _submit_command_and_assert_job_rejected(slurm_commands, submit_command_args):
+def _submit_command_and_assert_job_rejected(slurm_commands, submit_command_args, reason=None):
     """Submit a limit-violating job and assert the job is failed at submission."""
     result = slurm_commands.submit_command(**submit_command_args)
-    assert_that(result.stdout).contains("sbatch: error: Batch job submission failed:")
+    assert_that(result.stdout).contains(reason if reason else "sbatch: error: Batch job submission failed:")
```

Comment:
nit: you could set reason="sbatch: error: Batch job submission failed:" as a default argument in the signature

Reply:
Good idea! Changed.